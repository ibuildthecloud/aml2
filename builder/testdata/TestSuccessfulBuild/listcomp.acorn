[for x in list {
	x: 1
}, 2]