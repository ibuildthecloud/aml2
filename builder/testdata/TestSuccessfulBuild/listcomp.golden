&eval.Struct{Fields: []eval.Field{
	&eval.Embedded{Expression: &eval.For{
		Value: "x",
		Collection: &eval.Lookup{
			Pos: eval.Position{
				Filename: "listcomp.acorn",
				Offset:   10,
				Line:     1,
				Column:   11,
			},
			Key: "list",
		},
		Body: &eval.Struct{Fields: []eval.Field{&eval.KeyValue{
			Key: eval.FieldKey{
				Key: eval.Value{Value: value.String("x")},
				Pos: eval.Position{
					Filename: "listcomp.acorn",
					Offset:   18,
					Line:     2,
					Column:   2,
				},
			},
			Value: eval.Value{Value: value.Number("1")},
			Pos: eval.Position{
				Filename: "listcomp.acorn",
				Offset:   18,
				Line:     2,
				Column:   2,
			},
		}}},
	}},
}}
